### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
in this branch we:

- Initialized the project with components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable*/
```
Comment (elmar8287) [inline src/App.js]: - [x] Please, consider removing the linter disables from all the code of your app - let linter makes its direct job: find bugs on early steps of the development 👍

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Book = ({ title, author }) => (
+  <>
```
Comment (elmar8287) [inline src/components/Book.js]: - [x] Please, could you describe from which component these props are coming? Please, consider fixing this issue, to make your app work correctly, as required 👍

File: src/components/BookList.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import React from 'react';
+import AddBook from './BookInput';
+
+const BookList = () => (
+  <div className="book-list">
+    <AddBook />
+  </div>
+)
+
+export default BookList;
```
Comment (elmar8287) [inline src/components/BookList.js]: - [x] I think here you are missing the `Book` component, imported here, and the props which I talked about before should come from here, aka BookList => props => Book

Don't you think?

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```
Comment (elmar8287) [inline src/App.js]: - [x] Please, consider implementing the router, as requered: 

- Has two `Route` s, one for the `BookList` component and the other for the `Categories` component
In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

```
    <Route exact path="/">
      <BookList />
    </Route>
```

You have to implement the navigation: 

- Should have 2 `<Link>`s in the navbar which leads to the booklist and categories locations

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Kwaleyela-Ikafa.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good PR description 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
- Please, consider in the future giving the title for commits with the Capital letter - it will make your commits history looks professional

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Kwaleyela-Ikafa, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**